time/cookie mismatch upstream

Currently `just test` on a checkout without an existing `Cargo.lock` fails with

```rust
   Compiling devise_codegen v0.4.2
error[E0061]: this method takes 2 arguments but 1 argument was supplied
   --> /root/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/cookie-0.18.1/src/parse.rs:226:27
    |
226 |     let mut date = format.parse(s.as_bytes())?;
    |                           ^^^^^-------------- argument #2 of type `Option<Parsed>` is missing
    |
note: method defined here
   --> /root/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/time-0.3.52/src/parsing/parsable.rs:62:12
    |
62  |         fn parse(&self, input: &[u8], defaults: Option<Parsed>) -> Result<Parsed, error::Parse> {
    |            ^^^^^
help: provide the argument
    |
226 |     let mut date = format.parse(s.as_bytes(), /* Option<Parsed> */)?;
    |                                             ++++++++++++++++++++++

   Compiling derive_more-impl v2.1.1
For more information about this error, try `rustc --explain E0061`.
error: could not compile `cookie` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: recipe `test` failed with exit code 101
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,6 +16,12 @@ members = [
     "utoipa-actix-web",
 ]
 
+# cookie 0.18.1 (via rocket_http) does not compile against time 0.3.52+
+# (Parsable::parse gained a defaults argument). Cap until cookie releases a fix.
+# Keeps `just test` / fresh resolves working without committing Cargo.lock.
+[workspace.dependencies]
+time = { version = ">=0.3, <0.3.52" }
+
 [workspace.metadata.publish]
 order = [
     "utoipa-config",

--- a/examples/actix-web-multipart/Cargo.toml
+++ b/examples/actix-web-multipart/Cargo.toml
@@ -4,6 +4,8 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+# cookie (actix/rocket) does not build with time 0.3.52+; drop when cookie is fixed
+time = { version = ">=0.3, <0.3.52" }
 actix-web = "4"
 actix-multipart = "0.7"
 utoipa = { path = "../../utoipa", features = ["actix_extras"] }

--- a/examples/actix-web-multiple-api-docs-with-scopes/Cargo.toml
+++ b/examples/actix-web-multiple-api-docs-with-scopes/Cargo.toml
@@ -11,6 +11,8 @@ authors = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+# cookie (actix/rocket) does not build with time 0.3.52+; drop when cookie is fixed
+time = { version = ">=0.3, <0.3.52" }
 actix-web = "4"
 serde = { version = "1.0", features = ["derive"] }
 env_logger = "0.11.0"

--- a/examples/actix-web-scopes-binding/Cargo.toml
+++ b/examples/actix-web-scopes-binding/Cargo.toml
@@ -5,6 +5,8 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+# cookie (actix/rocket) does not build with time 0.3.52+; drop when cookie is fixed
+time = { version = ">=0.3, <0.3.52" }
 actix-web = "4"
 serde = { version = "1.0", features = ["derive"] }
 env_logger = "0.11.0"

--- a/examples/generics-actix/Cargo.toml
+++ b/examples/generics-actix/Cargo.toml
@@ -11,6 +11,8 @@ authors = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+# cookie (actix/rocket) does not build with time 0.3.52+; drop when cookie is fixed
+time = { version = ">=0.3, <0.3.52" }
 actix-web = "4"
 env_logger = "0.10.0"
 geo-types = { version = "0.7", features = ["serde"] }

--- a/examples/raw-json-actix/Cargo.toml
+++ b/examples/raw-json-actix/Cargo.toml
@@ -11,6 +11,8 @@ authors = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+# cookie (actix/rocket) does not build with time 0.3.52+; drop when cookie is fixed
+time = { version = ">=0.3, <0.3.52" }
 actix-web = "4"
 env_logger = "0.10.0"
 serde_json = "1.0"

--- a/examples/rocket-todo/Cargo.toml
+++ b/examples/rocket-todo/Cargo.toml
@@ -9,6 +9,8 @@ authors = ["Elli Example <example@example.com>"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+# cookie (actix/rocket) does not build with time 0.3.52+; drop when cookie is fixed
+time = { version = ">=0.3, <0.3.52" }
 rocket = { version = "0.5", features = ["json"] }
 utoipa = { path = "../../utoipa", features = ["rocket_extras"] }
 utoipa-swagger-ui = { path = "../../utoipa-swagger-ui", features = [

--- a/examples/simple-x-extensions/Cargo.toml
+++ b/examples/simple-x-extensions/Cargo.toml
@@ -5,6 +5,8 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+# cookie (actix/rocket) does not build with time 0.3.52+; drop when cookie is fixed
+time = { version = ">=0.3, <0.3.52" }
 actix-web = "4"
 env_logger = "0"
 utoipa = { path = "../../utoipa", features = [ "yaml" ] }

--- a/examples/todo-actix/Cargo.toml
+++ b/examples/todo-actix/Cargo.toml
@@ -9,6 +9,8 @@ authors = ["Example <example@example.com>"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+# cookie (actix/rocket) does not build with time 0.3.52+; drop when cookie is fixed
+time = { version = ">=0.3, <0.3.52" }
 actix-web = "4"
 serde = { version = "1.0", features = ["derive"] }
 env_logger = "0.11"

--- a/utoipa-gen/Cargo.toml
+++ b/utoipa-gen/Cargo.toml
@@ -51,7 +51,7 @@ smallvec = { version = "1.15", features = ["serde"] }
 rust_decimal = { version = "1", default-features = false }
 bigdecimal = { version = "0.4", default-features = false }
 chrono = { version = "0.4", features = ["serde"] }
-time = { version = "0.3", features = ["serde-human-readable"] }
+time = { workspace = true, features = ["serde-human-readable"] }
 jiff = { version = "0.2", features = ["serde"] }
 serde_with = "3.14"
 insta = { version = "1.47", features = ["json"] }

--- a/utoipa-rapidoc/Cargo.toml
+++ b/utoipa-rapidoc/Cargo.toml
@@ -15,6 +15,10 @@ rust-version.workspace = true
 features = ["actix-web", "axum", "rocket"]
 rustdoc-args = ["--cfg", "doc_cfg"]
 
+[features]
+# Pull capped `time` with rocket so cookie 0.18.1 cannot resolve time 0.3.52+.
+rocket = ["dep:rocket", "dep:time"]
+
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
@@ -23,6 +27,7 @@ utoipa = { version = "5.0.0", path = "../utoipa", default-features = false, feat
 ] }
 actix-web = { version = "4", optional = true, default-features = false }
 rocket = { version = "0.5", features = ["json"], optional = true }
+time = { workspace = true, optional = true }
 axum = { version = "0.8.4", default-features = false, features = [
     "json",
 ], optional = true }

--- a/utoipa-redoc/Cargo.toml
+++ b/utoipa-redoc/Cargo.toml
@@ -15,6 +15,10 @@ rust-version.workspace = true
 features = ["actix-web", "axum", "rocket"]
 rustdoc-args = ["--cfg", "doc_cfg"]
 
+[features]
+# Pull capped `time` with rocket so cookie 0.18.1 cannot resolve time 0.3.52+.
+rocket = ["dep:rocket", "dep:time"]
+
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
@@ -23,6 +27,7 @@ utoipa = { version = "5.0.0", path = "../utoipa", default-features = false, feat
 ] }
 actix-web = { version = "4", optional = true }
 rocket = { version = "0.5", features = ["json"], optional = true }
+time = { workspace = true, optional = true }
 axum = { version = "0.8.4", default-features = false, optional = true }
 
 [dev-dependencies]

--- a/utoipa-scalar/Cargo.toml
+++ b/utoipa-scalar/Cargo.toml
@@ -15,6 +15,10 @@ rust-version.workspace = true
 features = ["actix-web", "axum", "rocket"]
 rustdoc-args = ["--cfg", "doc_cfg"]
 
+[features]
+# Pull capped `time` with rocket so cookie 0.18.1 cannot resolve time 0.3.52+.
+rocket = ["dep:rocket", "dep:time"]
+
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
@@ -23,6 +27,7 @@ utoipa = { version = "5.0.0", path = "../utoipa", default-features = false, feat
 ] }
 actix-web = { version = "4", optional = true, default-features = false }
 rocket = { version = "0.5", features = ["json"], optional = true }
+time = { workspace = true, optional = true }
 axum = { version = "0.8.4", default-features = false, optional = true }
 
 [dev-dependencies]

--- a/utoipa-swagger-ui/Cargo.toml
+++ b/utoipa-swagger-ui/Cargo.toml
@@ -21,6 +21,8 @@ vendored = ["dep:utoipa-swagger-ui-vendored"]
 cache = ["dep:dirs", "dep:sha2"]
 minified = []
 gzip = ["dep:flate2"]
+# Pull capped `time` with rocket so cookie 0.18.1 cannot resolve time 0.3.52+.
+rocket = ["dep:rocket", "dep:time"]
 
 [dependencies]
 mime_guess = { version = "2.0" }
@@ -28,6 +30,8 @@ actix-web = { version = "4", optional = true, default-features = false, features
     "macros",
 ] }
 rocket = { version = "0.5", features = ["json"], optional = true }
+# See workspace.dependencies — only enabled with `rocket` feature.
+time = { workspace = true, optional = true }
 axum = { version = "0.8.4", default-features = false, features = [
     "json",
 ], optional = true }

